src/App.tsx

```diff
@@ -1,34 +1,60 @@
-import { RouterProvider, createBrowserRouter } from 'react-router-dom'
-import { ForgetPassword, Home, Login, Register, ResetPassword } from './Pages'
-import { AuthLayout, MasterLayout, NotFound } from './Components'
-
-
-
-  function App() {
-
-const routes = createBrowserRouter([
-  {
-    path: '/', element: <AuthLayout />, errorElement: <NotFound />, children: [
-      { index: true, element: <Login /> },
-      { path: "login", element: <Login /> },
-      { path: "register", element: <Register /> },
-      { path: "forgetPassword", element: <ForgetPassword /> },
-      { path: "resetPassword", element: <ResetPassword /> },
-    ]
-  },
-  {
-    path: 'dashboard', element: <MasterLayout />, errorElement: <NotFound />, children: [
-      { index: true, element: <Home /> },
-    ]
-  },
-])
-
-return <>
-  <RouterProvider router={routes} />
-</>
-
-
-
+/** @format */
+
+import { Provider } from "react-redux";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import { AuthLayout, MasterLayout, NotFound } from "./Components";
+import {
+  AddNewRoom,
+  Ads,
+  Bookings,
+  ForgetPassword,
+  Home,
+  Login,
+  Register,
+  ResetPassword,
+  Rooms,
+  Users,
+} from "./Pages";
+import { store } from "./Redux/Store";
+
+function App() {
+  const routes = createBrowserRouter([
+    {
+      path: "dashboard",
+      element: <MasterLayout />,
+      errorElement: <NotFound />,
+      children: [
+        { index: true, element: <Home /> },
+        { path: "users", element: <Users /> },
+        { path: "rooms", element: <Rooms /> },
+        { path: "addNewRoom", element: <AddNewRoom /> },
```

Comment:
It is recommended for better search engine optimization and readability to separate words in URLs with hyphens rather than using capital letters.
You can read more about that at this [StackOverflow Answer](https://stackoverflow.com/a/18450653/21382803)